test/e2e/specs/editor/various/preferences.spec.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'preferences', () => {
+	test( 'remembers sidebar dismissal between sessions', async ( {
+		page,
+		admin,
+	} ) => {
+		await admin.createNewPost();
+		// Open by default.
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Post' );
+
+		// Change to "Block" tab.
+		await page.click( 'role=button[name="Block"i]' );
+
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Block' );
+
+		// Regression test: Reload resets to document tab.
+		//
+		// See: https://github.com/WordPress/gutenberg/issues/6377
+		// See: https://github.com/WordPress/gutenberg/pull/8995
+		await page.reload();
+		await page.waitForSelector( '.edit-post-layout' );
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Post' );
+
+		// Dismiss.
+		await page.click(
+			'.edit-post-sidebar__panel-tabs [aria-label="Close settings"]'
```

Comment:
This one can be a role locator too:
![Screen Shot 2022-09-15 at 5 13 42 pm](https://user-images.githubusercontent.com/677833/190365246-214bca42-ad7c-4711-8b84-236885bb3bc0.png)

( button / "Close settings" )